ivy/functional/frontends/tensorflow/general_functions.py

```diff
@@ -461,3 +461,20 @@ def norm(tensor, ord="euclidean", axis=None, keepdims=None, name=None):
     "float32",
     "float64",
 )
+
+
+@to_ivy_arrays_and_back
+def approx_top_k(input, k=1, name=None):
+    input = ivy.array(input)
+    batch_size = input.shape[0]
+    num_classes = input.shape[1]
+
+    # Approximate the top k values
+    flattened_values = ivy.reshape(input, [batch_size * num_classes])
```

Comment:
just wondering, are the reshapes necessary or can it be detected through the shape and pass `axis=<dim_for_op>` in `ivy.top_k`?